app/views/tags/cardNewsAndUpdates.scala.html

```diff
@@ -30,6 +30,8 @@
     </@headingTag>
     <ul>
         <li><a class="ga-track-anchor-click"  href="@controllers.routes.InterstitialController.displayNewsAndUpdates()#verify" data-ga-event-category="link - click" data-ga-event-action="Income" data-ga-event-label="HMRC Personal Tax Account to stop using GOV.UK Verify">@messages("label.stop_using_Verify")</a></li>
+        <br>
```

Comment:
That `<br>` tag should not be here I think.

Reply:
I agree

Reply:
Fixed it now , thank you both